src/main/kotlin/blackjack/view/OutputView.kt

```diff
@@ -0,0 +1,20 @@
+package blackjack.view
+
+import blackjack.domain.Player
+import blackjack.domain.Players
+
+object OutputView {
+
+    fun outputPlayers(players: Players) {
+        println("${players.joinToString { it.name }}에게 ${players.size}장을 나누었습니다.")
+        players.forEach(::showPlayerCard)
+    }
+
+    fun showPlayerCard(player: Player) {
+        println("${player.name}카드: ${player.playerCards.joinToString { it.toString() }}")
```

Comment:
아래의 코드로 개선해 보면 어떨까요?

```suggestion
        println("${player.name}카드: ${player.playerCards.joinToString()}")
```